mandatory/1-fix-functions.js

```diff
@@ -21,32 +19,24 @@ function mood() {
 }
 
 function greaterThan10(num) {
-  let isBigEnough;
-
-  if (isBigEnough) {
+  if (num > 10) {
```

Comment:
This is correct but I think the task wanted the isBigEough variable to be assigned the comparison expression result.

ie. let isBigEnough = num > 10;